server/backend/db_backend.go

```diff
@@ -649,6 +654,53 @@ func (self *MongoBackend) getStats() *models.Stats {
 	}
 	return s
 }
+func (self *MongoBackend) getSignerStatsForRange(rangeDays int, nodes map[common.Address]models.Node) []models.SignerStats {
```

Comment:
This would be more testable if it accepted a `time.Time` and a `time.Duration`. It would also allow the call to `getBlockRange()` to use the exact same range.
```go
getSignerStatsForRange(end time.Time, dur time.Duration, ...
getBlockRange(end time.Time, dur time.Duration, ...
```